### ISSUE TITLE
Trouble Running Unity After Use

### ISSUE THREAD

**Issue by djfreeman84** [FunkyFr3sh/GameDVR_Config#9]
I downloaded and ran this yesterday trying to fix an issue with my microsoft game bar. Unfortunately it didn't fix the issue (not your fault) but since I ran the software I'm having trouble running Unity. I can't confirm this is more than coincidence, but it seems like it is. Opening Unity now uses up 100% of my CPU, slows everything down, and worse, makes it so that when I test play my game it is running at, like 2 fps. I'm trying to figure out if there is a way to roll back whatever changes I might have made running this software to determine if it is causing the problem. Help!

**Comment by FunkyFr3sh** [comment]
check this: https://github.com/FunkyFr3sh/GameDVR_Config/issues/5

**Comment by djfreeman84** [comment]
At first I had been trying to fix a problem with my game screen capture functionality, and downloaded your software as one of multiple possible solutions. That's when the problem started.

I am not able to confirm or deny that your software caused the problem - since I attempted multiple fixes. I also did not use your solution to fix the problem. I ended up reinstalling my graphics driver, and that fixed it, so I didn't get around to trying your solution.

But regardless, the problem is fixed, so mark this one resolved! Hooray!

Now if I could only figure out why my screen capture functionality wasn't working in the first place. *sigh*
